Procfile

```diff
@@ -0,0 +1 @@
+web: FLASK_APP=app.py python -m flask run --host=0.0.0.0 --port=$PORT
```

Comment:
@iTaylorCodes what is the use of this set up?

Reply:
This Procfile was for Heroku deployment, but I was creating the environments incorrectly. Removed file.